### PR TITLE
edgeTransport 2.0 release

### PR DESCRIPTION
**Pull request by johannah-pik**
## Purpose of this PR
Adjusting REMIND to function with the new edgeTransport 2.0 version:
Changes include:
-Change transport scenario names in sets and scenario config
-Remove files/scenarios no longer needed
-Include new transport input data files
-Adjust function calls
-Change package dependencies

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [ ] Bug fix 
- [x] Refactoring
- [ ] New feature 
- [ ] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [ ] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [x ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):
This PR will be merged with several other PRs related to the switch to edgeTransport 2.0:

- edgeTransport [#266](https://github.com/pik-piam/edgeTransport/pull/266/files)
- reporttransport [#18](https://github.com/pik-piam/reporttransport/pull/18)
- mrremind [#542](https://github.com/pik-piam/mrremind/pull/524)
- remind [#1749](https://github.com/remindmodel/remind/pull/1749)
- remind2 [#642](https://github.com/pik-piam/remind2/pull/624)
- piamInterfaces [#331](https://github.com/pik-piam/piamInterfaces/pull/331)

* Test runs are here: /p/projects/edget/Refactoring/remind/output
* Comparison of results (what changes by this PR?):

### CODE REVIEW

**Review by jmuessel — APPROVED** [review]

File: () [hunk 1]
```diff
@@ -0,0 +1,18 @@
+update edge_esm
+add!
+
+ jPlease enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# On branch develop
```
Comment (LaviniaBaumstark) [inline ():7]: where is this from?

File: config/default.cfg [hunk 2]
```diff
@@ -27,10 +27,10 @@ cfg$regionmapping <- "config/regionmappingH12.csv"
 ### Additional (optional) region mapping, so that those validation data can be loaded that contain the corresponding additional regions.
 cfg$extramappings_historic <- ""
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- "6.79"
+cfg$inputRevision <- "6.83edgeTransportReloaded"
```
Comment (LaviniaBaumstark) [inline config/default.cfg:30]: this input data sound abit like test-data. We do not have input data with a "better" version number, right?

File: core/sets.gms [hunk 3]
```diff
@@ -76,7 +76,7 @@ gdp_SSP2EU_NAV_lce "NAVIGATE demand scenarios: Low consumption energy (act + tec
 gdp_SSP2EU_NAV_all "NAVIGATE demand scenarios: All measures (ele + act + tec)"
 gdp_SSP2EU_CAMP_weak   "CAMPAIGNers scenario with low ambition lifestyle change"
 gdp_SSP2EU_CAMP_strong "CAMPAIGNers scenario with high ambition lifestyle change"
-gdp_SSP2EU_demRedWeak
+gdp_SSP2EU_demRedStrong
```
Comment (LaviniaBaumstark) [inline core/sets.gms:79]: please add an explanation of this scenario

File: eval
Comment (LaviniaBaumstark) [inline eval:1]: what about this file?

File: eval.parent
Comment (LaviniaBaumstark) [inline eval.parent:1]: and this file?

File: modules/29_CES_parameters/calibrate/datainput.gms [hunk 4]
```diff
@@ -147,7 +147,7 @@ Parameter
 p29_trpdemand       "transport demand"
 /
 $ondelim
-$include "./modules/29_CES_parameters/calibrate/input/pm_trp_demand.cs4r"
+$include "./modules/29_CES_parameters/calibrate/input/f29_trpdemand.cs4r"
```
Comment (LaviniaBaumstark) [inline modules/29_CES_parameters/calibrate/datainput.gms:150]: you also renamed the file in fullREMIND, right?

File: renv.lock
Comment (LaviniaBaumstark) [inline renv.lock:1]: also not part of our Git repo

File: () (hunk 1 above)
Comment (johannah-pik) [reply ():7]: hmm i dont know.. maybe we can just delete it?

**Review by jmuessel — APPROVED** [review]

**Comment by orichters** [comment]
Sorry to say, but after this merge REMIND is dysfunctional:
```
2280123  1900,LAM,gdp_SSP2EU,gdp_SSP2EU_demRedStrong,ECEMF_HighEl_LifestCha,te_esdie_frgt_lo,0.000342571576087318
****                                               $170
**** LINE 258726 INCLUDE     /p/tmp/oliverr/remind-edget/modules/35_transport/edge_esm/input/f35_esCapCost.cs4r
**** LINE     22 INCLUDE     /p/tmp/oliverr/remind-edget/modules/35_transport/edge_esm/datainput.gms
**** LINE     22 INCLUDE     /p/tmp/oliverr/remind-edget/modules/35_transport/edge_esm/realization.gms
**** LINE     16 INCLUDE     /p/tmp/oliverr/remind-edget/modules/35_transport/module.gms
**** LINE     30 BATINCLUDE  /p/tmp/oliverr/remind-edget/modules/include.gms
                             %1  datainput
**** LINE   1777 INPUT       /p/tmp/oliverr/remind-edget/output/gamscompile/main_default.gms
**** 170  Domain violation for element
```
Please run `make test` (or ideally, `make test-full` for such a big change) before merging as indicated in the check list of your PR description.

**Pull request merged**